core/src/main/resources/defaults/blueprints/cdp-opdb-702.bp

```diff
@@ -91,6 +95,12 @@
           {
             "refName": "hbase-GATEWAY-BASE",
             "roleType": "GATEWAY",
+            "configs": [
+              {
+                "name": "hbase_client_config_safety_valve",
+                "value": "<property><name>phoenix.schema.isNamespaceMappingEnabled</name><value>true</value></property><property><name>phoenix.queryserver.withRemoteUserExtractor</name><value>true</value></property><property><name>phoenix.functions.allowUserDefinedFunctions</name><value>true</value></property><property><name>hbase.use.dynamic.jars</name><value>false</value></property>"
```

Comment:
These configs if they are necessary should be exposed in CM. Safety valves are being removed where possible. https://jira.cloudera.com/browse/CB-4191

Reply:
The hbase.dynamic.jars property patch has been sitting on gerrit for more than five weeks.
The phoenix.queryserver.withRemoteUserExtractor one is already merged for 7.1
The other two cannot be exposed, as we Phoenix and Hbase share configuration files, and we haven't got  solution for handling that yet.